drivers/src/persistent.rs

```diff
@@ -47,6 +48,9 @@ pub struct PersistentData {
 
     pub fuse_log: FuseLogArray,
     reserved5: [u8; memory_layout::FUSE_LOG_SIZE as usize - size_of::<FuseLogArray>()],
+
+    pub dpe: DpeInstance,
```

Comment:
Hmm. I just realized depending on DPE here has the possibility of impacting ROM code (although maybe not since ROM doesn't use it).

@korran do you have any concerns with this?

Also @sree-revoori1 can you double check that the ROM binary is the same with and without this change?

Reply:
It is not the same. I did cargo build with and without this change, and then did wc -c target/debug/caliptra-rom to find the number of bytes in the caliptra-rom binary. With this change, it is 7711896 bytes, without it is 7138048 bytes.

Reply:
Done.